skill-package/skill.json

```diff
@@ -83,9 +87,34 @@
+            "messaging",
+            "telegram",
+            "whatsapp",
+            "messenger",
+            "threema",
+            "friends",
+            "family",
+            "communication",
+            "open-source"
+          ],
+          "largeIconUri": "https://s3.amazonaws.com/CAPS-SSE/echo_developer/2160/d1c355937c434917ba2c32b6bc48b53f/APP_ICON_LARGE?versionId=AsSQNRCzUAcga2toF24fr9.e2LzRYwuH&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20201121T120520Z&X-Amz-SignedHeaders=host&X-Amz-Expires=86400&X-Amz-Credential=AKIAWBV6LQ4QHAYALYJ7%2F20201121%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=6d91af71360b3178bbcdd5353645983d2c631939f2736a3cfddb0f222889fbcf",
+          "name": "Telegram Connect",
+          "smallIconUri": "https://s3.amazonaws.com/CAPS-SSE/echo_developer/3e1c/cd806b19b17c4433b66c034a8152366f/APP_ICON?versionId=naNt5mIKiAEhDwxf0CH.LTvn5LIPHRNj&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20201121T120520Z&X-Amz-SignedHeaders=host&X-Amz-Expires=86400&X-Amz-Credential=AKIAWBV6LQ4QHAYALYJ7%2F20201121%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=989def74c55c2682ae95a827bf606e41f3fedece6fc14f0da3eef132f79d67f3",
+          "summary": "Telegram Connect integra Alexa con Telegram Messenger"
         }
       },
-      "testingInstructions": "In order to test this skill you need an Amazon Account with a phone number and a Telegram Messenger Account that is linked to that phone number.\n\nTo get a free phone number you can use:\nwww.textnow.com\nLogin: TODO\nPassword: TODO\n\nAdd this phone number to the Amazon testing account.\n\nCreate a Telegram Messenger account with you phone number:\n- Download the Telegram Messenger for your OS at: https://telegram.org/\n- At some point Telegram will sent you a code to your FreeNow number. Check your texts for that code.\n\nTest the skill\n- Alexa, open telegram connect\n- Couple Alexa with Telegram Messenger:\n        Alexa will ask you for a code. Check you Telegram Messenger for that code.\n- After you successfully coupled Alexa with Telegram, you can say:\n       Check my Telegrams\n       Learn more about the skill"
+      "testingInstructions": "Per testare questa skill avrai bisogno di un account Amazon con un numero di telefono collegato ed un account Telegram, collegato allo stesso numero.\n\nPer ottenere un numero di telefono gratuito, puoi usare:\nwww.textnow.com\nLogin: TODO\nPassword: TODO\n\nAggiungi questo numero al tuo account Amazon.\n\nCrea un account su Telegram con questo numero:\n- Scarica Telegram Messenger per il tuo sistema operativo qui: https://telegram.org/\n- Telegram invierà un codice al numero ottenuto con FreeNow. Controlla i messaggi.\n\nTesta la skill\n- Alexa, apri telegram connect\n- Integra Alexa con Telegram Messenger:\n        Alexa ti chiederà di un codice. Controlla ti sia arrivato in Telegram.\n- Dopo aver integrato Alexa e Telegram con successo, potrai dire:\n       Controlla i miei Telegram\n       Informazioni su questa skill"
```

Comment:
Please change the testing instructions back to English. We send them only once.

Reply:
Solved by [6be6fb3](https://github.com/LorenzHW/Telegram-Connect/pull/5/commits/6be6fb3e8832113d7544a631cf795a5e5798fbe1)